draft-ietf-quic-transport.md

```diff
@@ -4872,11 +4874,11 @@ max_ack_delay (0x0b):
 disable_active_migration (0x0c):
 
 : The disable active migration transport parameter is included if the endpoint
-  does not support active connection migration ({{migration}}). Peers of an
-  endpoint that sets this transport parameter MUST NOT send any packets,
-  including probing packets ({{probing}}), from a local address or port other
-  than that used to perform the handshake.  This parameter is a zero-length
-  value.
+  does not support active connection migration ({{migration}}) on the address
+  being used during the handshake.  Peers of an endpoint that sets this
+  transport parameter MUST NOT send any packets, including probing packets
+  ({{probing}}), from a local address or port other than that used to perform
+  the handshake. This parameter is a zero-length value.
```

Comment:
To me it seems that we need to also change this sentence (starting from "Peers ..."). We need to tighten the restriction to only when the client sends the packets to the original address of the server.

Maybe something like: _Peers of an endpoint that sets this transport parameter MUST NOT send any packets, including probing packets ({{probing}}) __to the address that was used to perform the handshake__, from a local address or port other than that used to perform the handshake._

Reply:
Good point (I made this same point verbally, but failed to notice the absence of a fix).

Reply:
I toyed with that and the existing sentence already has so many sub-phrases that it became too unwieldy.  I'll see if I can split it up any better.